app/models/comment.rb

```diff
@@ -11,4 +11,22 @@ def self.search(search)
       all
     end
   end
+
+  def created
```

Comment:
There is a gem that handles this (more or less correct) also internationalized. It's called timeago or so.
Please use this instead.